src/dialects/db2/query-generator.js

```diff
@@ -180,6 +181,11 @@ export class Db2QueryGenerator extends AbstractQueryGenerator {
   }
 
   describeTableQuery(tableName, schema) {
+    if (typeof tableName === 'object') {
```

Comment:
this could call the updated `extractTableDetails` function instead, but it's stuck inside of the changeColumns PR https://github.com/sequelize/sequelize/pull/14687/files#diff-54c14e4744281a47d62196b5892c4004fec6e2f3c4d90f444aa3bd0a880f2209R88

Reply:
Yeah, I took a quick look at that yesterday but came across some failing tests. Will look into this